Pages/OverlayPage.js

```diff
@@ -6,6 +6,9 @@ var OverlayPage = function() {
     var overlaySelf = $locator('.overlay-media');
     var overlayContentLink = $locator('.overlayLeftPanel .content-title-link');
     var overlayTwitterShare = $locator('.overlayLeftPanel .btn.share-twitter');
+    var TweetAuthorLeft = $locatorxpath("//span[contains(@class,'overlay-author')]");
```

Comment:
remove the $locatorxpath. Try using only css. Your css locator in this case is ".overlay-author"